examples/dreambooth/train_dreambooth_lora.py

```diff
@@ -1364,7 +1366,7 @@ def compute_text_embeddings(prompt):
         pipeline = pipeline.to(accelerator.device)
 
         # load attention processors
-        pipeline.load_lora_weights(args.output_dir)
+        pipeline.load_lora_weights(args.output_dir, weight_name="pytorch_lora_weights.bin")
```

Comment:
`"pytorch_lora_weights.bin"` is the default:

https://github.com/huggingface/diffusers/blob/8b18cd8e7f6be0cf2904dfec4285d4ba98c5586f/src/diffusers/loaders.py#L1215C1-L1219C47

So, safe to not add it explicitly.